code/game/objects/items/weaponry.dm

```diff
@@ -301,11 +301,11 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	embedding = list("embedded_pain_multiplier" = 8, "embed_chance" = 100, "embedded_fall_chance" = 0, "embedded_impact_pain_multiplier" = 15) //55 damage+embed on hit
 
 /obj/item/switchblade
-	name = "switchblade"
+	name = "long switchblade"

```

Comment:
Why the name change here?

Reply:
Differentiates it from the "iron switchblade" and "plastitanium switchblade" a bit more. Implies that the reason this switchblade does the highest amount of damage is because it's longer than the two new ones.